src/Symfony/Component/DependencyInjection/composer.json

```diff
@@ -22,7 +22,7 @@
     },
     "require-dev": {
         "symfony/yaml": "^4.4|^5.0",
-        "symfony/config": "^4.4|^5.0",
+        "symfony/config": "^5.0",
```

Comment:
Is there a way not doing this? We'd better resist these when possible.

Reply:
not sure how to detect Config 5.0 :/

then again .. it's nice to enforce the same behavior for everyone on DI 5.0 no?

Reply:
I would try to try harder here: shouldn't the test case be moved to the config component?
This doesn't look legit to me.

Reply:
updated

> shouldn't the test case be moved to the config component?

i favored a di+config integration test, instead of a config unit test. Integration test have to deal with low/high deps either way. It now covers the deps=high case, assuming the legacy test is covered on lower branches still.

Reply:
agree 4.4|5.0 is the better constraint, which is conceptually the same behavior already but allows users to opt for deprecations vs. exceptions basically.